docs/source/topics/proc_connecting-to-remote-instance.adoc

```diff
@@ -63,4 +63,4 @@ $ sudo systemctl reload NetworkManager
 $ oc login -u developer -p developer https://api.crc.testing:6443
 ----
 +
-The remote OpenShift Web Console is available at \https://console-openshift-console.apps-crc.testing.
+The remote {ocp} web console is available at \https://console-openshift-console.apps-crc.testing.
```

Comment:
so far it was `{openshift} web console`

Reply:
This change was based on how they write about the web console in the docs for OCP. There, they always mention OCP explicitly, and I assume that's the best practice for us as well.